app/assets/javascripts/app/account_transaction_report.coffee

```diff
@@ -0,0 +1,26 @@
+class AccountTransactionReport
+  constructor: (@$element) ->
+    return unless @$element.length > 0
+    @registerExportAllHandler()
+
+  registerExportAllHandler: ->
+    $('#js--exportAll').click (event) => @exportAllClicked(event)
+
+  exportAllClicked: (event) ->
+    newTo = prompt 'Have the report emailed to this address:', $('#js--toEmail').val()
+
+    if newTo
+      url = event.target.href
+      $.post(url, { to_email: newTo }, ->
```

Comment:
For consistency's sake, I think we should do an actual submit instead of AJAX similar to what we do for the export in the reports tab.